depthai_sdk/setup.py

```diff
@@ -0,0 +1,36 @@
+import io
+
+from setuptools import setup
+
+with open('requirements.txt') as f:
+    required = f.readlines()
```

Comment:
So on why they are fixed right now
- PyTube was proven to be instable since after one new release we couldn't download YT videos at all
- blobconverter is our tool, so we will try to point to latest version anyways, so fixing it here could update an existing installation if found
- OpenCV had issues with its latest versions not being correctly installed under different OSes, and we wanted to point users to the versions.

`depthai` is not fixed since I didn't want to change library version check right now and it was easier to have it "loose" in SDK and fixed in demo requirements.

I would say to leave it as is for now, since these are the exact requirements we had in the demo script, and for initial release it's ok. We should however check for the cases where opencv might be preinstalled or if we could update/loosen our current requirements.